### ISSUE TITLE
Add a new organization to ROR: 
 
Deutsches Zentrum für Integration- und Migrationsforschung

### ISSUE THREAD

**Issue by mariagould** [ror-community/ror-updates#468]
Summary of request: Add a new organization to ROR

Name of organization: 
ROR ID: 
Which part of the record needs to be changed? 
Description of change: 

Name of organization: Deutsches Zentrum für Integration- und Migrationsforschung
Website: https://www.dezim-institut.de/
Link to publications: https://onlinelibrary.wiley.com/doi/full/10.1002/per.2228  https://www.dezim-institut.de/fileadmin/Publikationen/Research_Notes/DeZIM_Research_Notes_05_210323_3_RZ_web.pdf
Organization type: Government (An organization that is part of or operated by a national or regional government and that conducts or supports research)
Wikipedia page: https://de.wikipedia.org/wiki/Deutsches_Zentrum_f%C3%BCr_Integrations-_und_Migrationsforschung
Wikidata ID: 
ISNI ID: 
GRID ID: 
Crossref Funder ID: 
Aliases: German Center for Integration and Migration Research
Labels:
Acronym/abbreviation: DeZIM
Related organizations: Bundesministerium für Familie, Senioren, Frauen und Jugend
City: Berlin
Country: Germany
Year established: 2017
How will a ROR ID for this organization be used? To identify affiliations for research publications, To identify affiliations for datasets, To acknowledge research funders, To be integrated in a scholarly publishing system
Other information about this request:

**Comment by adambuttrick** [comment]
German government organization focused on research related to migration issues. Independently publishes research reports on these topics and is used an affiliation on similar publications in other forums.

Google Scholar affiliation usage: https://scholar.google.com/citations?user=BhjnhlAAAAAJ
ORCID affiliation usage: https://orcid.org/0000-0003-2102-2527 ; https://orcid.org/0000-0002-7182-1792 ; https://orcid.org/0000-0001-7147-1609

**Comment by arthurpsmith** [comment]
Yes, this research institute looks like it meets the criteria for having a ROR id. Unfortunately the German wikipedia page and the wikidata item were deleted for lack of supporting info - I've asked for undeletion on Wikidata; I'll update here if something changes. Anyway, otherwise it should proceed.

**Comment by arthurpsmith** [comment]
Wikidata ID has been restored.

**Comment by adambuttrick** [comment]
Assigned ROR ID https://ror.org/03ym65z81 in release v1.0.
